explainaboard/metrics/f1_score.py

```diff
@@ -118,6 +123,61 @@ def calc_metric_from_aggregate(
         return f1
 
 
+@dataclass
+@register_metric_config
+class APEF1ScoreConfig(MetricConfig):
```

Comment:
This seems very similar or even identical to SeqF1Score, would it be possible to just use SeqF1Score instead? If there's only a small difference, then it seems that we could express that difference through a parameter for SeqF1Score instead of creating a new class.

Reply:
This task is kinda special compared with common sequence labeling tasks (such as NER), For example, this is one example's tags:
```
'tags': ['Review-O', 'Review-O', 'Review-O', 'Review-O', 'Review-O', 'Review-O', 'Review-O', 'Review-B-6', 'Review-I-6', 'Review-B-3', 'Review-B-5', 'Review-I-5', 'Review-I-5', 'Review-I-5', 'Review-B-7', 'Review-I-7', 'Review-I-7', 'Review-B-4', 'Review-B-2', 'Review-B-1', 'Review-B-8', 'Review-I-8', 'Review-I-8', 'Review-I-8', 'Reply-O', 'Reply-B-3', 'Reply-I-3', 'Reply-I-3', 'Reply-B-5', 'Reply-I-5', 'Reply-I-5', 'Reply-I-5', 'Reply-B-4', 'Reply-I-4', 'Reply-I-4', 'Reply-I-4', 'Reply-B-2', 'Reply-I-2', 'Reply-I-2', 'Reply-B-3', 'Reply-I-3', 'Reply-B-1']
```
(Review-B-5, Review-I-5, Review-I-5, Review-I-5, Reply-B-5, Reply-I-5, Reply-I-5, Reply-I-5) is one successful  identification (or we can say it is a span)
I couldn't find a good way to unify them at the moment, but feel free to raise a better solution.

Reply:
I'm OK with keeping it as-is for this PR and thinking of ways to generalize later. But could you add documentation to the code so it's clear what the difference is for people looking at it later (like you just explained here)?

Reply:
Sure and sounds good